frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -98,38 +98,74 @@ const pageDescription =
   'Catalog. Cluster admins can install additional apps which will show up here automatically.';
 
 // Filter property white list
-const filterGroups = ['kind'];
+const filterGroups = ['type'];
 
 const getAvailableFilters = (initialFilters) => {
   const filters = _.cloneDeep(initialFilters);
-  filters.kind = {
-    ClusterServiceClass: {
-      label: 'Service Class',
-      value: 'ClusterServiceClass',
+  filters.type = {
+    operatorBacked: {
+      label: 'Operator Backed',
+      active: true,
+      kinds: {
+        ClusterServiceVersion: {
+          label: 'Installed Operators',
+          value: 'InstalledOperator',
+          active: false,
+        },
+      },
+    },
+    helm: {
```

Comment:
Do not add the helm filter here as there is no helm integration till now. My PR adds the filter for that - https://github.com/openshift/console/pull/3850

Reply:
I could remove this. But the filters in the referenced PR wouldn't work with type integration.

Reply:
This works perfectly with the existing item structure. Hence, if the items with kind `HelmChart` are added, it will fit in without the need to make any changes in the filters.

Reply:
I don't see a point adding a filter that doesn't have any item in it.

Reply:
Just part of the UX for accommodating future needs, IMO.

Reply:
Well, I think those future needs should be handled by the future story that adds that feature in future. That is actually part of the acceptance criteria for the Helm stories and not part of this one. Even thought the mockups show that in this one as well. We can clarify this with UX. 
cc: @serenamarie125

Reply:
I've put out Helm from the filters